codeforamerica/vita-min
# When MFS, Spouse name & SSN should not be populated in the return header or spouse fields on pdf

## Link to pivotal/JIRA issue
- [link](https://codeforamerica.atlassian.net/browse/FYST-1396?atlOrigin=eyJpIjoiMjFlNDEyMGQ0MjI2NGNjMzk1ZDdjZWQzZmE0YjYzN2QiLCJwIjoiaiJ9)
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- prevent spouse info population in return header or pdfs for all states if filing status is married-filing-separately
## How to test?
- Select a MFS test case and check that secondary filer information is not included in the XML or PDF
